source/controls.js

```diff
@@ -83,6 +85,64 @@ chrome.runtime.onMessage.addListener(request => {
   }
 })
 
+const updateMediaSessionState = () => {
+  if ('mediaSession' in navigator) {
+    let api = window.wrappedJSObject.externalAPI;
+    navigator.mediaSession.playbackState = api.isPlaying() ? 'playing' : 'paused';
+
+    const track = api.getCurrentTrack();
+    const urlTpl = 'https://' + state.cover;
+    const artworks = ['100x100', '200x200', '400x400'];
```

Comment:
Please replace `const` with `let`. It’s OK to declare `const` on the top level scope, but it might be confusing inside of methods, because it’s not a true constant. I’ll try to find the article I refer

Reply:
Fixed